aliases/available/general.aliases.bash

```diff
@@ -57,11 +62,16 @@ alias ipy='ipython'
 alias piano='pianobar'
 
 alias ..='cd ..'         # Go up one directory
+alias cdh='cd ~/'  # Go in home
 alias cd..='cd ..'       # Common misspelling for going up one directory
 alias ...='cd ../..'     # Go up two directories
 alias ....='cd ../../..' # Go up three directories
 alias -- -='cd -'        # Go back
 
+function cdls() {
```

Comment:
Same here, please move this to the `base` plugin.

Reply:
[jusdepatate@962f309](https://github.com/jusdepatate/bash-it/commit/962f3093a2fa8c364e72d8e970ce7e349e8caa54)